# Loop Info
Header header

int32 robot0
int32 robot1
int32 index0
int32 index1
float32 noise
int32 init_yaw
geometry_msgs/Transform pose0
geometry_msgs/Transform pose1
sensor_msgs/PointCloud2 scan_cloud
geometry_msgs/Transform pose_between